dash/backend/src/modules/shared/services/email.service.ts

```diff
@@ -23,13 +24,17 @@ export class EmailService {
 
       if (this.configService.get('email.smtp.host')) {
         await this.mailerService.sendMail(mailData); // send async
```

Comment:
Does this service return true/false based on whether it was successful sending the email?

Reply:
I believe it returns true when the email is sent.